### PR TITLE
updated screenshots and docs

### PR DESCRIPTION
**Pull request by tronxd**

### CODE REVIEW

File: docs/Introduction/Getting Started.md [hunk 1]
```diff
@@ -15,15 +15,33 @@ checkov -d /user/tf
...
 
 ```bash
-Passed Checks: 1, Failed Checks: 1, Suppressed Checks: 0
+Passed checks: 7, Failed checks: 15, Skipped checks: 2
+
+Check: "S3 Bucket has an ACL defined which allows public access."
+	PASSED for resource: aws_s3_bucket.sls_deployment_bucket_name
+	File: /../regionStack/main.tf:23-32
+
+Check: "Ensure the S3 bucket has access logging enabled"
```
Comment (schosterbarak) [inline docs/Introduction/Getting Started.md]: not intresting  enough

File: docs/Introduction/Getting Started.md [hunk 2]
```diff
@@ -15,15 +15,33 @@ checkov -d /user/tf
...
+	FAILED for resource: aws_s3_bucket.template_bucket
+	File: /main.tf:81-92
+
+		81 | resource "aws_s3_bucket" "template_bucket" {
+		82 |   region        = var.region
+		83 |   bucket        = local.bucket_name
+		84 |   # checkov:skip=CKV_AWS_19:Honestly dear, I don't give a damn
```
Comment (schosterbarak) [inline docs/Introduction/Getting Started.md]: description that is logical

**Comment by schosterbarak** [comment]
maybe we should retake it with slower scroll?

**Comment by schosterbarak** [comment]
can you please update https://github.com/bridgecrewio/checkov/blob/update-docs/docs/Advanced%20concepts/Suppressions.md too?

File: docs/Introduction/Getting Started.md [hunk 3]
```diff
@@ -80,17 +225,23 @@ from checkov.terraform.models.enums import CheckResult, CheckCategories
 from checkov.terraform.checks.resource.base_check import BaseResourceCheck
```
Comment (schosterbarak) [inline docs/Introduction/Getting Started.md]: Let start the sample policy section with terraform snippet before the python snippet.
This way the user will know adding a policy starts at a terraform code base that we would like to scan

**Comment by schosterbarak** [comment]
mind updating: https://github.com/bridgecrewio/checkov/blob/update-docs/README.md too?

**Comment by schosterbarak** [comment]
do we have a sample of junitxml output in the docs?

**Comment by tronxd** [comment]
> do we have a sample of junitxml output in the docs?

https://github.com/bridgecrewio/checkov/blob/update-docs/docs/Introduction/Results.md

**Pull request merged**